proto/schema.proto

```diff
@@ -0,0 +1,170 @@
+syntax="proto3";
+
+option go_package = "github.com/kubernetes-csi/external-snapshot-metadata/pkg/grpc";
+
```

Comment:
Could/Should we put a comment attribution to `github.com/container-storage-interface/spec/csi.proto`?